### ISSUE TITLE
Bug importing to 2 different store views

### ISSUE THREAD

**Issue by xmarkclx** [bigbridge-nl/product-import#19]
Hi Patrick! Awesome plugin btw.

When I do:

```
        <store_view code="lt_spare_1">
            <delete code="status"/>
        </store_view>

        <store_view code="lt_spare_2">
            <delete code="status"/>
        </store_view>
```

I get an error:
```
Error in line 10: Element 'lt_spare_2': This element is not expected.
```

**Comment by garfix** [comment]
Hello Mark,
Thanks for the compliment! This keeps me going :)

Sorry for the late reply. And I do not have an answer right away. The error you get seems to be an xml validation error that you used `lt_spare_2` as an element, but it is an attribute value. Your xml seems fine. You try to remove the status attribute values on the store view level.

Can you show me the xml of the complete product? You can obfuscate the actual product data. Also on which line of that xml does the error really apply?

**Comment by patrick-bigbridge** [comment]
Hi Mark,

You ran into a bug in the XSD that validates the XML. It did not allow multiple store_view elements.

It took me some time to realize that the XML structure that I had imagined could not be strictly enforced by the XML Schema standard. I did not want to change the structure, because that could introduce backward incompatibilities and trouble developers, so I loosened the XSD a bit. As a consequence of this, It is now possible to have multiple <global> elements as wel as multiple <store_view>, and that should not really be possible, but this should not cause any trouble.

In short: what you want to achieve should work now. It's in release 1.5.1
